protocol/protocol.proto

```diff
@@ -1005,4 +1008,7 @@ enum Err {
     MINIPOOL_MISSING_EVENTS = 60;
     STREAM_LAST_BLOCK_MISMATCH = 61;
     DOWNSTREAM_NETWORK_ERROR = 62;
+
+    // Stream replica on the local node does not have miniblock with given number yet. 
```

Comment:
We can mention that this is expected to be a temporary error and that the client can retry it.